src/ui/public/chrome/directives/app_switcher_link/app_switcher_link.less

```diff
@@ -22,18 +23,19 @@
       text-decoration: none;
     }
   }
+
+  .is-app-links-wrapper-open & {
+    width: @as-open-width;
+  }
```

Comment:
This is a pretty confusing use of a nested selector. Couldn't it just be a separate rule with a selector like `.is-app-links-wrapper-open .app-switcher-link`